prog/bootstrap_rhizome.rb

```diff
@@ -29,6 +27,10 @@ def user
 echo #{sshable.keys.map(&:public_key).join("\n").shellescape} | sudo tee /home/rhizome/.ssh/authorized_keys > /dev/null
 SH
 
-    push Prog::InstallRhizome, {"target_folder" => frame["target_folder"]}
+    push Prog::InstallRhizome, {"target_folder" => frame["target_folder"]}, next_label: "finish"
```

Comment:
Wew it's slightly brutal how many strings are in "push", and that we don't have a nice way to avoid free-handing errors in next_label or a way to check them on file load. But, that's for another day. Maybe let's pencil in these renovations for the next quarter. It's one of those sub-legible tasks that's easy to forget. Just planting the seed of a thought here.

Reply:
I created a issue for it https://github.com/ubicloud/ubicloud/issues/1293

Budding and pushing with metaprogramming might be interesting